guides/common/modules/proc_creating-a-remote-source-for-a-custom-file-type-repository.adoc

```diff
@@ -12,46 +12,42 @@ ifndef::satellite[]
 * You have a server running {EL} 7 or 8 registered to your {Project}.
 endif::[]
 ifdef::satellite[]
-* You have a server running {EL} 7 or 8 registered to your {Project} or the Red{nbsp}Hat CDN.
+* You have a server running {EL} 8 registered to your {Project} or the Red{nbsp}Hat CDN.
 * Your server has an entitlement to the {RHELServer} and {ProjectName} Utils repositories.
 endif::[]
 * You have installed an HTTP server.
-ifndef::orcharhino[]
-For more information about configuring a web server, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/deploying_different_types_of_servers/setting-apache-http-server_deploying-different-types-of-servers[Setting up the Apache HTTP web server] in {RHEL}{nbsp}8 _Deploying different types of servers_ or https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/system_administrators_guide/ch-web_servers#s1-The_Apache_HTTP_Server[The Apache HTTP Server] in the {RHEL}{nbsp}7 _System Administrator's Guide_.
+ifdef::satellite[]
```

Comment:
This seems to be exclusive to Satellite.

Reply:
Agreed.